pkg/compose/logs_test.go

```diff
@@ -172,7 +172,7 @@ func TestComposeService_Logs_ServiceFiltering(t *testing.T) {
 
 type testLogConsumer struct {
 	mu sync.Mutex
-	// logs is keyed container; values are log lines
+	// logs is keyed containerType; values are log lines
```

Comment:
nit: over-aggressive find/replace

```suggestion
	// logs is keyed by container ID; values are log lines
```